### PR TITLE
Add google maps api key to build command for deployment

### PR DESCRIPTION
**Pull request by toddjordan**
This was brought up in slack the other day that the google maps key that we had the tutorial followers set up earlier was not being propagated to deployments.  Also threw in a blurb for windows users to partially address #2003

### CODE REVIEW

File: source/tutorial/deploying.md [hunk 1]
```diff
@@ -26,28 +47,23 @@ Then you can use the `surge` command to deploy your application.
...
 use a custom domain that you own and have pointed the DNS to one of surges servers.
 If the second argument is left blank surge will prompt you with a suggested subdomain.
 
-To deploy to the same URL after making changes, perform the same steps, reusing 
+To deploy to the same URL after making changes, perform the same steps, reusing
 the same domain as before.
 
 ```shell
 rm -rf dist
-ember build --environment=development
```
Comment (toddjordan) [inline source/tutorial/deploying.md]: oops, gotta clean this up first. to re-run the build (with the key)

Comment (locks) [reply source/tutorial/deploying.md]: PR not ready?

Comment (toddjordan) [reply source/tutorial/deploying.md]: yeah, just need to add back build with key here.

**Comment by toddjordan** [comment]
@locks forgot about this one.  I believe its good to go now...

**Pull request merged**